cdc/generation.cc

```diff
@@ -298,17 +298,6 @@ bool should_propose_first_generation(const gms::inet_address& me, const gms::gos
     });
 }
 
-future<db_clock::time_point> get_local_streams_timestamp() {
-    return db::system_keyspace::get_saved_cdc_streams_timestamp().then([] (std::optional<db_clock::time_point> ts) {
-        if (!ts) {
-            auto err = format("get_local_streams_timestamp: tried to retrieve streams timestamp after bootstrapping, but it's not present");
-            cdc_log.error("{}", err);
```

Comment:
Wouldn't it still be valuable to at least have a log message that this unexpected situation has happened so that we know the CDC management tables need to be checked and maybe fixed?

Reply:
Done